airbyte-cdk/python/airbyte_cdk/sources/file_based/file_based_stream_reader.py

```diff
@@ -92,6 +92,13 @@ def filter_files_by_globs_and_start_date(self, files: List[RemoteFile], globs: L
                     seen.add(file.uri)
                     yield file
 
+    def file_size(self, file: RemoteFile) -> int:
+        """
+        Utility method to get size of the read file, is required for stream writer implementation
+        return size in bytes unit
+        """
+        return 0
+
```

Comment:
Wdyt of making this abstract so that connectors have to implement it (aka so they don't forget to implement it) when moving to the latest CDK version.

Maybe something like this:

```suggestion
    @abstractmethod
    def file_size(self, file: RemoteFile) -> int:
        """Utility method to get size of the remote file.
        
        This is required for connectors that will support writing to
        files. If the connector does not support writing files, then the 
        subclass can simply `return 0`.
        """
        ...

```

Reply:
It was like that, but Lint or Mypy was yelling at me for something; let me retry.

Reply:
Ok, done after fixing some unit tests that were required to implement the new abstract method file_size.